packages/kmod-6.1-nvidia/kmod-6.1-nvidia.spec

```
@@ -178,19 +208,32 @@ install -p -m 0644 firmware/gsp_tu10x.bin %{buildroot}%{_cross_libdir}/firmware/
 
 popd
 
+# Begin NVIDIA fabric manager
+pushd fabricmanager-linux-%{fm_arch}-%{tesla_ver}-archive
+install -p -m 0755 bin/nv-fabricmanager %{buildroot}%{_cross_libexecdir}/nvidia/tesla/bin
+install -p -m 0755 bin/nvswitch-audit %{buildroot}%{_cross_libexecdir}/nvidia/tesla/bin
+
+install -d %{buildroot}%{_cross_datadir}/nvidia/nvswitch
```

the topologies need to go in the tesla sub-directory:
```suggestion
install -d %{buildroot}%{_cross_datadir}/nvidia/tesla/nvswitch
```